src/user/user.service.ts

```diff
@@ -43,26 +75,26 @@ export class UserService {
 
       return userProfile;
```

Comment:
is this route intended for only owner of profile, or public?

Reply:
do you think we need this API? since we have get user API where profile included, Front-end might not use this API maybe delete?